pkgs/applications/audio/vital/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, unzip
+
+, alsa-lib
+, libjack2
+, curl
+, xorg
+, libGL
+, freetype
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vital";
+  version = "1.5.5";
+
+  unpackCommand = ''
+    mkdir -p root
+    unzip $curSrc root
+  '';
+
+  src = fetchurl {
```

Comment:
How about using `fetchzip`?

Reply:
Ohhhh right, can't believe I didn't notice this